subprojects/docs/src/docs/release/notes.md

```diff
@@ -21,9 +21,35 @@ With [dependency constraints](userguide/managing_transitive_dependencies.html#se
 
 In the example, the version of `commons-codec` that is brought in transitively is `1.9`. With the constraint, we express that we need at lease `1.11` and Gradle will now pick that version during dependency resolution.
 
-### Advanced POM support (preview)
+### Improved POM support: optional dependencies
```

Comment:
I'd probably leave out the colon and make it one distinct title e.g. "Support for optional dependencies in POM consumption".

Reply:
Same comment for the other titles below.

Reply:
Right - that's probably more consistent. I changed the titles.